src/main/java/org/reaktivity/nukleus/http_cache/internal/HttpCacheCounters.java

```diff
@@ -60,5 +62,7 @@ public HttpCacheCounters(
         this.promises = supplyCounter.apply("http-cache.promises");
         this.cacheEntries = supplyAccumulator.apply("http-cache.cache.entries");
         this.requestGroups = supplyAccumulator.apply("http-cache.request.groups");
+        this.cacheHitFrequencies = supplyAccumulator.apply("http-cache.cache.hit.frequencies");
```

Comment:
Is this actually counting cache hit frequencies? or just the number of cache frequency buckets we have in total?

Reply:
Just the number of cache frequency buckets we have in total